atlasdb-remoting-api/src/main/java/com/palantir/atlasdb/config/RemotingClientConfigs.java

```diff
@@ -21,8 +21,5 @@ private RemotingClientConfigs() {
         // Constants
     }
 
-    public static final RemotingClientConfig ALWAYS_USE_CONJURE = ImmutableRemotingClientConfig.builder()
```

Comment:
note to self/you: you'll have to change the places where this is referred to explicitly namely big internal product and it's related projects.

Reply:
Yeah, that's right. Thanks